server/game/cards/12-2T2D/AlexanderSequoia.js

```diff
@@ -0,0 +1,30 @@
+const GameActions = require('../../GameActions/index.js');
+const DudeCard = require('../../dudecard.js');
+
+class AlexanderSequoia extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.isInTownSquare(),
+            effect: ability.effects.modifySundownDiscard(1)
+        });
+        
+        this.action({
+            title: 'Noon',
+            playType: 'noon',
+            ifCondition: () => this.isInTownSquare(),
+            ifFailMessage: context => {
+                this.game.addMessage('{0} uses {1}\'s ability but {1} does not unboot because he is not in town square', context.player, this);
+            },
+            cardCondition: this.booted,
```

Comment:
Add also `actionContext: { card: this, gameAction: 'unboot' }`. Just in case there will be some effect introduced that prevents unbooting.
```suggestion
            cardCondition: this.booted,
            actionContext: { card: this, gameAction: 'unboot' },
```